Bottom: 5017d3a2e22b200daa0b7a75dc87cff7c2581bd5
Top:    7e7f9b6ecb0f50bc42af6fc7b8bdb34b01e0dc03
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-02 14:28:23 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!






---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 2dcf4b8..5e56536 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -312,13 +312,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
